Project.toml

```diff
@@ -11,6 +10,9 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StatsModels = "3eaba693-59b7-5ba5-a881-562e759f1c8d"
 
+[compat]
+StatsModels = "0.5.0"
```

Comment:
Can we change this to "< 0.6.0"? If I'm reading it correctly, the current version spec `= "0.5.0"` wouldn't allow to run with a version below 0.5.

Reply:
Never mind, we'll leave it as is for now.